sky/tools/release_engine.py

```diff
@@ -47,31 +47,34 @@ def __init__(self, category, name):
 
 ARTIFACTS = {
     'android-arm': [
-        Artifact('shell', 'SkyShell.apk'),
-        Artifact('shell', 'flutter.mojo'),
-        Artifact('shell', 'libflutter_library.so'),
+        Artifact('shell', 'chromium-debug.keystore'),
+        Artifact('shell', 'icudtl.dat'),
+        Artifact('shell', 'dist/shell/SkyShell.apk'),
+        Artifact('shell', 'dist/shell/flutter.mojo'),
+        Artifact('shell', 'dist/shell/libflutter_library.so'),
+        Artifact('shell', 'gen/sky/shell/shell/classes.dex'),
+        Artifact('shell', 'gen/sky/shell/shell/shell/libs/armeabi-v7a/libsky_shell.so'),
     ],
     'linux-x64': [
-        Artifact('shell', 'icudtl.dat'),
-        Artifact('shell', 'sky_shell'),
-        Artifact('shell', 'sky_snapshot'),
-        Artifact('shell', 'flutter.mojo'),
-        Artifact('shell', 'libflutter_library.so'),
+        Artifact('shell', 'dist/shell/icudtl.dat'),
+        Artifact('shell', 'dist/shell/sky_shell'),
+        Artifact('shell', 'dist/shell/sky_snapshot'),
+        Artifact('shell', 'dist/shell/flutter.mojo'),
+        Artifact('shell', 'dist/shell/libflutter_library.so'),
     ]
 }
 
 
 def upload_artifacts(dist_root, config, commit_hash):
     for artifact in ARTIFACTS[config]:
-        src = os.path.join(artifact.category, artifact.name)
         dst = GS_URL % {
             'category': artifact.category,
             'config': config,
             'commit_hash': commit_hash,
-            'name': artifact.name,
+            'name': os.path.basename(artifact.path),
         }
         z = ','.join([ 'mojo', 'dat' ])
```

Comment:
Maybe add "so" and "dex" to this list?  This is the list of extensions that get compression.  (gsutil is nutty)